src/nvim/option_defs.h

```diff
@@ -401,6 +401,7 @@ EXTERN int p_cst;               // 'cscopetag'
 EXTERN long p_csto;             // 'cscopetagorder'
 EXTERN long p_cspc;             // 'cscopepathcomp'
 EXTERN int p_csverbose;         // 'cscopeverbose'
+EXTERN int p_clbg;            // 'ctrldisambig'
```

Comment:
Perhaps a flag in 'cpoptions' instead? It is useful to know all of the compatibility options, without having to scour the documentation. Having one option with flags helps discoverability, instead of numerous one-off options.

Reply:
This PR was superseded by #17825, which enables these unconditionally.